pkg/cli/mirror/create_test.go

```diff
@@ -132,3 +179,19 @@ func TestCreateWithCancel(t *testing.T) {
 
 	require.Equal(t, true, opts.interrupted)
 }
+
+func GenerateConfigPath(path string, baseFile string, targetFile string) {
```

Comment:
Nit: This function could be private if it will just be used in this test file

Reply:
Ack. Changed to private.